app/src/Homepage/Homepage.css

```diff
@@ -6,6 +6,9 @@
 .header {
   background-color: #6100AC;
   height: 11vh;
+  top: '0px';
```

Comment:
Is there a reason why top is 0px? I would think that is the default. If it is needed, why ' ' around them? Does the style guide say to do that..?

Reply:
My bad, the 0px is left over from when I was experimenting with different positions than fixed and now has no impact. The quotes are just copying the style from the article I was referencing that used inline CSS, and are also unnecessary. I can remove. Also, we don't have a CSS style guide, so maybe we should look into that.